docs/resources/workflow_task_archive_slack_channels.md

```diff
@@ -29,7 +29,6 @@ resource "rootly_workflow_task_archive_slack_channels" "archive_slack_channels"
   skip_on_failure = false
   enabled         = true
   task_params {
-    name = "Archive Slack channels"
     channels {
       id   = "{{ incident.slack_channel_id }}"
       name = "{{ incident.slack_channel_id }}"
```

Comment:
@yjperez channel name needs to be `{{ incident.slack_channel_name }}` not `{{ incident.slack_channel_id }}` I believe.

Reply:
So I tried using the incident `incident.slack_channel_name` but it overrides the `incident.slack_channel_id` and it is required to have both. So if we are going to be opinionated, I feel it is better to use the ID over the name

Reply:
oh that feels like a bug ha